tests/run-coverage/uses_crate.rs

```diff
@@ -1,8 +1,12 @@
 // FIXME #110395
-// ignore-llvm-cov-show-diffs
+// ignore-linux
```

Comment:
I'm not sure this has the same effect as the test will still run on some platforms like macOS and Windows. Should this just be `// ignore` for now?

Reply:
Ah yes, this deserves more explanation.

I’ve verified that these tests actually pass in CI on macOS and Windows. It seems that they are only broken on Linux, but the makefile didn’t have a way to ignore a single tests on a single platform, so when they broke on Linux they ended up being disabled on all platforms.

(To be honest, I changed this to `ignore-linux` because I didn’t know about plain `ignore`. I do think it’s fine to re-enable the tests on other platforms, but I’m willing to change it if you still think this is a bad idea.)

Reply:
For more context, these tests seem to have been broken/disabled by #110393, which is kind of baffling and I don’t know if anyone has had a chance to properly investigate what happened. I suspect it has something to do with optimisation.

Reply:
Yeah, I was also confused to see that the ignore came from #110393. If these tests pass on `!linux`, then by all means, let's run them!